package/src/common/configure.ts

```diff
@@ -55,17 +55,17 @@ export async function configure(
 
   // record dev config
   const devConfig = createDevConfig(
-    Deno.env.get("DENO") || "",
-    Deno.env.get("DENO_DOM") || "",
-    Deno.env.get("PANDOC") || "",
-    Deno.env.get("DARTSASS") || "",
-    Deno.env.get("ESBUILD") || "",
+    Deno.env.get("QUARTO_DENO") || "",
+    Deno.env.get("QUARTO_DENO_DOM") || "",
+    Deno.env.get("QUARTO_PANDOC") || "",
+    Deno.env.get("QUARTO_DART_SASS") || "",
+    Deno.env.get("QUARTO_ESBUILD") || "",
     config.directoryInfo.bin,
   );
   writeDevConfig(devConfig, config.directoryInfo.bin);
-  info("");
+  info("Wrote dev config to bin directory");
 
-  if (Deno.build.os !== "windows") {
+  if (Deno.build.os !== "windows" && Deno.env.get("QUARTO_NO_SYMLINK") !== undefined) {
```

Comment:
The `QUARTO_NO_SYMLINK` behavior may be reversed here unless I'm misunderstanding it.

Reply:
In that case since they have different uses I think we should keep `QUARTO_SHARE_DIR` and `QUARTO_SHARE_PATH` exactly as they are